JAVA1/src/main/Main.java

```diff
@@ -0,0 +1,43 @@
+package main;
+import Solution.FileSearchInHomeDirectory;
+import Utils.utils;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    /**
+     * Continuously asks user to enter a regex pattern and prints matching files in home directory
+     * @param args not used
+     * @throws IOException can throw this exception
+     */
+    public static void main(String args[])throws IOException {
+
+        String s;
+        Scanner input = new Scanner(System.in);
+        FileSearchInHomeDirectory path = new FileSearchInHomeDirectory();
```

Comment:
name "path" specifies path for something. Make sure name used for object precisely describe their use.